UPDATING.md

```diff
@@ -41,6 +41,10 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Deprecated airflow.utils.file.TemporaryDirectory
+
+Instead user should use `tempfile.TemporaryDirectory` which provides the same functionality.
```

Comment:
Can you add example?

Reply:
I think now the description is sufficient.